src/libraries/MessagesLib.sol

```diff
@@ -504,6 +506,18 @@ library MessagesLib {
         return (_msg.toBytes32(1));
     }
 
+    /**
+    * Recover Tokens sent to the wrong contract
+    *
+    * 0: call type (uint8 = 1 byte)
+    * 1-20: The contract address (address = 20 bytes)
+    * 21-40: The recipient address (address = 20 bytes)
```

Comment:
These should be 32 bytes, of which the first 20 bytes are parsed as an address, for compatibility with future non EVM use cases